src/lib/testHelperShared.js

```diff
@@ -1,3 +1,4 @@
+require("./projectRequireHack");
```

Comment:
projectRequireHack requires runThroughBabel which requires babel-polyfill. The next line also requires babel-polyfill. Isn't this going to error out?